schema.sql

```diff
@@ -1,5 +1,14 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
+USE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+    id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
```

Comment:
👏👏👏👏 Awesome job making your `ID` field `NOT NULL` and `auto increment`.

Reply:
Thank you.

Reply:
@brytebee , please review again?